src/public/components/Admin.js

```diff
@@ -68,7 +68,7 @@ export default class Admin extends React.Component {
   async componentWillMount() {
     await setInterval(async () => {
       await this.getAttendance();
-    }, 3000)
+    }, 30000)
```

Comment:
Changed it to 30 secs so when i was testing it i dont see get request every 3 sec on my terminal